docker-service/templates/docker-compose-service

```diff
@@ -16,6 +16,7 @@
 
 start()
 {
+    /usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml pull
```

Comment:
`up` will pull or make any required image. why the separate `pull`?

Reply:
it won't actually.
if a certain tag is already present 'latest' or 'whatever' docker up will use the local image it has on the host but never fetch the latest registry upstream image for that tag.